deployments/container/nvidia-mig-manager-example-hopper-blackwell.yaml

```diff
@@ -276,11 +276,17 @@ data:
             "7g.94gb": 1
     
       # GB200 HGX
-      all-1g.24gb.me:
+      all-1g.23gb:
```

Comment:
So L279 and L245 are duplicates now. Let's remove L245. We don't need the device-filter for `all-1g.23gb` anymore

Reply:
@tariq1890 Please see if my latest push is ok now